packages/portal-proto/src/components/DropdownWithIcon/DropdownWithIcon.tsx

```diff
@@ -79,19 +85,24 @@ export const DropdownWithIcon = ({
   zIndex = undefined,
   customDataTestId = undefined,
   tooltip = undefined,
+  customBorderStyling = "",
 }: DropdownWithIconProps): JSX.Element => {
+  const [trapped, setTrapped] = useState(false);
+  const focusTrap = useFocusTrap(trapped);
```

Comment:
did you forget to remove these?

Reply:
i noticed when i removed these, the focus doesn't highlight the first element in the menu dropdown

<img width="212" alt="Screenshot 2024-01-04 at 2 54 29 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/90918464/3181e389-b003-498f-aff7-6189ce37e5f1">
vs. 
<img width="201" alt="Screenshot 2024-01-04 at 2 54 09 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/90918464/0c889e69-8359-4c90-9107-96e021e88f5f">